src/resources/ApiKeys/ApiKeysInterfaces.ts

```diff
@@ -169,6 +170,33 @@ export interface DuplicateApiKeyOptions {
     lifetimeDuration: string;
 }
 
+export interface ApiKeyListOptions {
+    /**
+     * Filter API keys based on their status.
+     * @see {@link ApiKeyStatusFilter} for possible values
+     */
+    status?: ApiKeyStatusFilter;
+}
+
+/**
+ * Represents the different statuses an API key can be filtered by.
+ * @enum {string}
+ */
+export enum ApiKeyStatusFilter {
+    /** Keys that are currently active and usable */
+    ACTIVE = 'ACTIVE',
+    /** Keys that will be disabled in the near future */
+    SOON_TO_BE_DISABLED = 'SOON_TO_BE_DISABLED',
+    /** Keys that will expire in the near future */
+    SOON_TO_BE_EXPIRED = 'SOON_TO_BE_EXPIRED',
+    /** Keys that are active but have been flagged as exposed */
+    ACTIVE_AND_EXPOSED = 'ACTIVE_AND_EXPOSED',
+    /** Keys that have been deactivated */
+    DEACTIVATED = 'DEACTIVATED',
+}
```

Comment:
This enum should go in `Enum.ts`, we had problem in the past with enums not being exported when exported in `Interfaces` files